### PR TITLE
Tests update

### PR DESCRIPTION
**Pull request by mbulatova**
1. Fix split test. In our testing system ports 2-3 are slower than
0-1 (different NICs). Use in test only equivalent ports 0 and 1 for
both send/receive.
2. Add latency test using perf_main.go.
3. Add recieve/sent ratio pass condition.
4. Some refactor

### CODE REVIEW

File: test/stability/test_split/test-split-part1.go [hunk 1]
```diff
@@ -102,46 +98,52 @@ func main() {
...
 }
 
+// Function to use in generator
+func generatePacket(pkt *packet.Packet, context flow.UserContext) {
+	atomic.AddUint64(&count, 1)
+	if count%5 == 0 {
+		generatePacketGroup1(pkt, context)
+	} else {
+		generatePacketGroup2(pkt, context)
+	}
```
Comment (ifilippov) [inline test/stability/test_split/test-split-part1.go]: Like Separate here

File: test/stability/test_split/test-split.conf [hunk 2]
```diff
@@ -3,4 +3,3 @@
 #  Source address, Destination address, L4 protocol ID, Source port, Destination port, Output port
         ANY            ANY              	ANY            ANY            111              1
         ANY            ANY              	ANY            ANY            222              2
-        ANY            ANY              	ANY            ANY            333              3
```
Comment (ifilippov) [inline test/stability/test_split/test-split.conf]: No need of removing this

File: test/performance/latency/latency-part1.go [hunk 3]
```diff
@@ -34,7 +34,7 @@ var (
 	BINS         int    = 10
 	SKIP_NUMBER  uint64 = 100000
 	SPEED        uint64 = 1000000
-	PASSED_LIMIT uint64 = 90
```
Comment (mbulatova) [inline test/performance/latency/latency-part1.go]: Yes we can achieve higher rate 84-91%. To do that
1. generated packets should be diverse
2. sent packet number should be increased (total sent ~10 mln)
I will update test now

Comment (ifilippov) [inline test/performance/latency/latency-part1.go]: Correct this please

Comment (mbulatova) [inline test/performance/latency/latency-part1.go]: Done

**Comment by ifilippov** [comment]
You can commit

**Pull request merged**